packages/compass-settings/src/utils/user-preferences.ts

```diff
@@ -0,0 +1,33 @@
+import { promisifyAmpersandMethod } from 'mongodb-data-service';
+import Preferences from 'compass-preferences-model';
+import type { THEMES } from 'compass-preferences-model';
+
+export type UserPreferences = {
+  autoUpdates: boolean;
+  enableMaps: boolean;
+  trackErrors: boolean;
+  trackUsageStatistics: boolean;
+  enableFeedbackPanel: boolean;
+  theme: THEMES.DARK | THEMES.LIGHT | THEMES.OS_THEME;
+};
```

Comment:
Let's include also networkTraffic here.

Reply:
We can defer this to the actual ticket for making networkTraffic work as expected, and we’ll be moving this list to compass-preferences-model anyway.